src/components/Navigation/Navigation.test.jsx

```diff
@@ -28,6 +28,12 @@ describe("Navigation", () => {
     expect(navigationDisplay.length).toBe(1);
   });
 
+  test("renders AppBar display", () => {
+    const wrapper = setup();
+    const appBarDisplay = findByTestAttribute(wrapper, "app-bar-display");
+    expect(appBarDisplay.length).toBe(1);
```

Comment:
❓ Can you try `expect(appBarDisplay.exists()).toBeTruthy();`?

Reply:
Addressed: commit d432723ed287368994edd0de8aae243c7d9ed62e